docker/development-easy-macos/README.md

```diff
@@ -0,0 +1,14 @@
+### Easy Development Docker Environment for macOS with M1, M2, M* SoCs
+The instructions for The Easy Development Docker environment can be found at [CONTRIBUTING.md](../../CONTRIBUTING.md#code-contributions-local-development).
+
+#### File Permissions
+
+UIDs and GIDs are handled differently between macOS and Linux. This discrepancy will cause permission errors on .git/objects/pack directory
+ files similar to:
+
+```
+development-easy-macos-openemr-1     | chown: /var/www/localhost/htdocs/openemr/.git/objects/pack/pack-ab1e32999500a2e52117c503b8126b3f77e3ff26.pack: Permission denied
+```
```

Comment:
Is this permission error happening when using git on the host outside the docker? (or is it happening when working inside the docker?)

Reply:
Hi I've been trying and failing to get openEMR running locally on macOS Ventura. I've been receiving that same permission error repeatedly. When I execute the `opener-cmd up` command, nothing happens in my terminal, though Docker Desktop shows those containers are running with following permission errors.  

I've tried using `docker-compose up`. I receive the same error but it is continuously printed in the terminal till I quit exit the command manually.

```
development-easy-openemr-1     | chown: /var/www/localhost/htdocs/openemr/.git/objects/pack/pack-a673fdb18ea8eb86a4f3bab0bbbe51d8b633a070.idx: Permission denied
development-easy-openemr-1     | chown: /var/www/localhost/htdocs/openemr/.git/objects/pack/pack-a673fdb18ea8eb86a4f3bab0bbbe51d8b633a070.rev: Permission denied
development-easy-openemr-1     | chown: /var/www/localhost/htdocs/openemr/.git/objects/pack/pack-a673fdb18ea8eb86a4f3bab0bbbe51d8b633a070.pack: Permission denied
development-easy-openemr-1 exited with code 0
```

`openemr-cmd`:
<img width="425" alt="Screenshot 2023-11-16 at 10 50 57 AM" src="https://github.com/openemr/openemr/assets/63010977/39633fe9-eed0-48c7-9262-1729f579f88e">

`docker-compose up`:
<img width="1271" alt="Screenshot 2023-11-16 at 10 52 56 AM" src="https://github.com/openemr/openemr/assets/63010977/6396fb2f-34b3-499c-b12e-16370a15a123">
<img width="1010" alt="Screenshot 2023-11-16 at 10 53 10 AM" src="https://github.com/openemr/openemr/assets/63010977/d7155734-c0a3-47c1-b41d-36bc1ec044ce">

Reply:
> Is this permission error happening when using git on the host outside the docker? (or is it happening when working inside the docker?)

The permissions error is in the Docker container on the first run.

Specifically, we map host files from .git to OpenEMR container file paths and attempt to run chown on them. However, macOS and Linux UID/GIDs are not the same, and the script in the container will throw permission errors, as @toduyemi pointed out.

My quick and dirty way to set overly permissive permissions, but there has to be a more elegant solution.